app/models/manageiq/providers/google/container_manager/refresh_worker/watch_thread.rb

```diff
@@ -0,0 +1,15 @@
+class ManageIQ::Providers::Google::ContainerManager::RefreshWorker::WatchThread < ManageIQ::Providers::Kubernetes::ContainerManager::RefreshWorker::WatchThread
+  private
+
+  def noop?(notice)
+    notice.object&.kind == "Endpoints" && filter_endpoint?(notice.object)
+  end
+
+  def filter_endpoint?(endpoint)
+    # The base kubernetes parser uses the endpoint subset addresses and targetRefs
+    # to build "container_groups_refs" in order to link pods to container_services
+    #
+    # If an endpoint doesn't have any subsets then it is a pointless update
+    endpoint.subsets.blank?
+  end
```

Comment:
Should this just belong in the base kubernetes code?  It seems the same argument in the comment would apply there as well, right?

Reply:
Yeah maybe?  I'll put in a PR to move it

Reply:
https://github.com/ManageIQ/manageiq-providers-kubernetes/pull/443